examples/scalar_data/eyja_volcano.py

```diff
@@ -15,19 +15,19 @@
 
 
 def main():
-    # Create a Stamen terrain background instance.
-    stamen_terrain = cimgt.Stamen('terrain-background')
+    # Create a background image tile.
+    google_terrain = cimgt.GoogleTiles(style="satellite")
 
     fig = plt.figure()
 
     # Create a GeoAxes in the tile's projection.
-    ax = fig.add_subplot(1, 1, 1, projection=stamen_terrain.crs)
+    ax = fig.add_subplot(1, 1, 1, projection=google_terrain.crs)
 
     # Limit the extent of the map to a small longitude/latitude range.
     ax.set_extent([-22, -15, 63, 65], crs=ccrs.Geodetic())
 
     # Add the Stamen data at zoom level 8.
```

Comment:
```suggestion
    # Add the Google data at zoom level 8.
```
Also in the module docstring.

Reply:
Oops, thanks! Just pushed an update with both of those.

Reply:
This line still says "Stamen"... 👀

Reply:
Uggh, sorry! Not enough coffee yet apparently.